src/components/ScrollGradient/ScrollGradient.stories.js

```diff
@@ -98,4 +99,32 @@ storiesOf(components('ScrollGradient'), module)
       `,
       },
     }
-  );
+  )
+  .add('with internal gradients',
+  () => (
+    <ScrollGradient
+      color={color('color', gradientColor)}
+      style={style}
+      className={className}
+    >
+      <p>Content before the scrollable element.</p>
+
+      <ScrollGradient
+        color={color('child color (color)', gradientColor)}
+        style={{ height: '6rem', margin: '1rem 0'}}
+      >
+        <p>Does not need to scroll so does not have a gradient.</p>
+      </ScrollGradient>
```

Comment:
For this example I tried to see what happens in this case if the `>` child combinater you added is removed, to see if a scroll gradient is incorrectly applied here. But I didn't see any extra scroll gradients 🤔 

<img width="932" alt="Screen Shot 2020-03-25 at 12 11 32 PM" src="https://user-images.githubusercontent.com/9057921/77565148-ce610700-6e91-11ea-9925-c31b4706e79f.png">


Am I testing this correctly? Could you show me how to recreate the bug? Thank you!

Reply:
Hi there! Thanks for the prompt review 😄. Here is what I did to do test the as-is:
1. I turned off the CSS changes I made
2. I used the knobs to change both scroll gradient colors to some obnoxious ones that stand out more
3. Scrolled to notice that when the gradients on the parent appear, so does the child's ones.
![Kapture 2020-03-25 at 13 35 01](https://user-images.githubusercontent.com/5973294/77573537-ce1b3880-6e9e-11ea-890e-afe994e25196.gif)

To test the PR (to-be), added the CSS changes back in and made sure to keep the obnoxious gradient colors:
![Kapture 2020-03-25 at 13 47 20](https://user-images.githubusercontent.com/5973294/77573792-2fdba280-6e9f-11ea-85c8-1f3272049b88.gif)

Notice how the child scroll gradient is not shown so they are independent of each other.

Reply:
Thank you!